students/ScottL/lesson04/mailroom_part2.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+
+#-------------------------------------------------#
+# Title: Mailroom Part-2 Assignment
+# Dev: Scott Luse
+# Date: Feb 10, 2018
+#-------------------------------------------------#
+
+
+
+# -- Data --#
+DONOR_HISTORY_DICT = {
```

Comment:
You have this capitalized like its a constant, but the dict will always be changing.  Better to have as lower case.